Cargo.lock

```diff
@@ -2725,9 +2779,9 @@ checksum = "c7d73b3f436185384286bd8098d17ec07c9a7d2388a6599f824d8502b529702a"
 
 [[package]]
 name = "libp2p"
-version = "0.28.1"
+version = "0.24.0"
```

Comment:
this seems to be an accidental downgrade when resolving merge conflicts

Reply:
I went through just now and, in every case where `Cargo.lock` showed a downgrade relative to master, manually reset it to the newer version. Then I did `cargo check --all`, and all my edits got reverted. Wherever those new versions were specified--mainly `libp2p-*`--they no longer appear to be.

Reply:
Pushed cf2062a. I just did `git checkout origin/master Cargo.lock && cargo check --workspace`. Hopefully, this will fix it.

Reply:
That commit causes test failures. I don't know _why_ it does--the mechanism of the failure isn't obvious--but it does seem to.

Reply:
Hmm, I think a merge with master might resolve it, let me try.